src/gfnff/gfnff_ini.f90

```diff
@@ -1849,6 +1849,30 @@ integer function itabrow6(i)
       write(env%unit,'(10x,"#optfrag :",3x,i0)') topo%nfrag
       !write(env%unit,*) '#optfrag :',nsystem
 
+
+      ! check if triple bonded carbon is present (for torsion term)
+      nn=0
+      do i=1, mol%n
+        if (mol%at(i).eq.6.and.topo%nb(20,i).eq.2) then
+          do j=1, 2
+            nbi=topo%nb(j,i)
+            if (mol%at(nbi).eq.6.and.topo%nb(20,nbi).eq.2) then
+              nn = nn + 1
+            endif
+          enddo
+        endif
+      enddo
+      if (nn.ne.0) then
+        ! fix double counting
+        nn = nn/2
+        allocate(topo%sTorsl(6, nn), source=0)
+        call specialTorsList(nn, mol, topo, topo%sTorsl)
+      else
+        ! allocate with size() = 0
+        allocate(topo%sTorsl(6, nn), source=0)
```

Comment:
Do you really need this?
you can just remove this and then check in **gfnf_eg**, if the `topo%sTorsl` allocated

Reply:
I changed it below, thanks.